Sources/App/WebView/WebViewController.swift

```diff
@@ -1096,6 +1101,18 @@ extension WebViewController: WKScriptMessageHandler {
             }
         }
     }
+
+    private func threadCredentialsRequested() {
+        if #available(iOS 16.4, *) {
+            guard let server = Current.settingsStore.menuItemTemplate?.server else { return }
```

Comment:
`self.server` -- every WebViewController is bound to exactly 1 server, and it's non-optional